Sell/OrderingJourney/OrderingJourneyView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  OrderingJourneyView.swift
+//  RealifeTech
+//
+//  Created by Mickey Lee on 17/08/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import SwiftUI
+import WebKit
+
+public struct OrderingJourneyView: View {
+
+    @ObservedObject private var store = WebViewStore()
+    @State private var canGoBack = false
+    @State private var canGoForward = false
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+
+    var dismiss: (() -> Void)?
```

Comment:
[Note] This is only assigned from UIHostController as the presentationMode doesn't support dismissing from UIKit until iOS 15.
https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-15-beta-release-notes#Updates-in-iOS-&-iPadOS-15-beta
> isPresented, PresentationMode, and the new DismissAction action dismiss a hosting controller presented from UIKit. (52556186)